tests/js/jest-test-file-setup.js

```diff
@@ -75,7 +75,8 @@ global.wcSettings = {
 		failed: 'Failed',
 		paid: 'Paid',
 	},
-	l10n: {
+	locale: {
+		siteLocale: 'en_US',
 		userLocale: 'en_US',
```

Comment:
Updating this to the accurately match what is available in the `wcSettings` global JS variable.